ship.config.js

```diff
@@ -57,6 +57,10 @@ module.exports = {
   pullRequestTeamReviewers: ['frontend-experiences-web'],
   buildCommand: ({ version }) =>
     `NODE_ENV=production VERSION=${version} yarn build`,
+  // Make sure the local branch is in sync with remote
+  // before pushing the ref and the tag on it
+  afterPublish: ({ exec }) =>
+    exec('git fetch && git rebase origin/master master'),
```

Comment:
should this change maybe be made in shipjs itself?

Reply:
You're right, that would be better. I'll apply the fix there.